src/parse.c

```diff
@@ -1727,10 +1727,77 @@ handle_ovs_bond_lacp(yaml_document_t* doc, yaml_node_t* node, const void* data,
     return handle_netdef_str(doc, node, data, error);
 }
 
+static gboolean
+handle_ovs_bridge_bool(yaml_document_t* doc, yaml_node_t* node, const void* data, GError** error)
+{
+    if (cur_netdef->type != NETPLAN_DEF_TYPE_BRIDGE)
+        return yaml_error(node, error, "Key is only valid for iterface type 'openvswitch bridge'");
```

Comment:
Ah, indeed, you are right. Fixing.

Reply:
Oh sorry, you are totally right!
I was mixing it up with `handle_bond_mode`, which is a special case, as it can be applied to ovs or non-ovs bonds via the `parameters` mapping.